### PR TITLE
chore: rename export zip to `{flow-slug}-{session-id}.zip`

### PR DESCRIPTION
**Pull request by jessicamcinchak**
When downloading zips via "Send to email", councils want to more clearly see which service a zip is associated with for reporting. 

Relies on https://github.com/theopensystemslab/planx-core/pull/328

Testing:
- Run through a send to email flow using your own email as the `teams.submission_email` - confirm the zip is named as expected on download

### CODE REVIEW

**Comment by jessicamcinchak** [comment]
[Rename ZIP produced by Send to email to include service name](https://trello.com/c/yVmx7QZl/2846-rename-zip-produced-by-send-to-email-to-include-service-name)

**Review by DafyddLlyr — APPROVED** [review]
Nice and simple! ✅ 

Update: Jumped the gun here sorry, didn't realise it was still in draft 🤦

**Comment by jessicamcinchak** [comment]
Zip is working as expected on pizza here, but invite to pay is broken - 
https://api.2893.planx.pizza/invite-to-pay/9b02ef22-1cf6-4ec0-b602-c3650706b48a is throwing error `"could not initiate a payment request: Cannot read properties of undefined (reading 'id')"` 

Can't spot where I've missed a session type update, going to switch tasks for a bit and come back with fresh eyes later.

File: api.planx.uk/modules/send/utils/exportZip.ts [hunk 1]
```diff
@@ -199,11 +200,11 @@ export class ExportZip {
   filename: string;
   private tmpDir: string;
 
-  constructor(sessionId: string) {
+  constructor(sessionId: string, flowSlug: string) {
     this.zip = new AdmZip();
     // make a tmp directory to avoid file name collisions if simultaneous applications
     this.tmpDir = fs.mkdtempSync(path.join(os.tmpdir(), sessionId));
-    this.filename = path.join(__dirname, `ripa-test-${sessionId}.zip`);
+    this.filename = path.join(__dirname, `${flowSlug}-${sessionId}.zip`);
```
Comment (DafyddLlyr) [inline api.planx.uk/modules/send/utils/exportZip.ts:207]: This gives me the following on Pizza - 
image

Are you getting this also? Could we drop `dirname` here?

Comment (jessicamcinchak) [reply api.planx.uk/modules/send/utils/exportZip.ts:207]: `dirname` only occurs on docker envs as I understand it (so local & pizzas), should be automatically omitted on staging / prod !

Comment (jessicamcinchak) [reply api.planx.uk/modules/send/utils/exportZip.ts:207]: eg http://api.editor.planx.dev/admin/session/d4d9249a-dc1e-4996-9bfc-15fd3adab242/zip returns plain `ripa-test-{sessionId}.zip`

Comment (DafyddLlyr) [reply api.planx.uk/modules/send/utils/exportZip.ts:207]: Perfect, thanks for clarifying! Makes sense considering we don't get this on staging / prod.

**Review by DafyddLlyr — APPROVED** [review]
- ITP tested ✅ 
 - Send to email zip name tested ✅ 

All looking good!

**Pull request merged**